### ISSUE TITLE
No module named 'boto3' after 0.9.x release

### ISSUE THREAD

**Issue by ericmbarnard** [erm/mangum#113]
Awesome project!

We've been using this to run Starlette in Lambda. After installing the newer 0.9 version, I'm now getting the following stack trace:
```
  File "./<PROJECT>/_lambda.py", line 1, in <module>
    from mangum import Mangum
  File "/c/git/<PROJECT>/.venv/lib/python3.7/site-packages/mangum/__init__.py", line 1, in <module>
    from .adapter import Mangum
  File "/c/git/<PROJECT>/.venv/lib/python3.7/site-packages/mangum/adapter.py", line 13, in <module>
    from mangum.protocols.websockets import WebSocketCycle
  File "/c/git/<PROJECT>/.venv/lib/python3.7/site-packages/mangum/protocols/websockets.py", line 6, in <module>
    from mangum.websocket import WebSocket
  File "/c/git/<PROJECT>/.venv/lib/python3.7/site-packages/mangum/websocket.py", line 10, in <module>
    import boto3
ModuleNotFoundError: No module named 'boto3'
```

Looks like it's importing a websocket class from boto3 as part of the initial import path for the entire lib. If your intention is to have boto3 as a hard dependency of the project - that's cool, then I'm thinking you'll want to move boto3 out of the extras here: 
https://github.com/erm/mangum/blob/master/setup.py#L19

Our hope was that there would be a default or slim option for this lib to keep dependencies low and help us maximize lambda startup times. If there's a way to preserve a non-boto3 option for this library, we'd sure appreciate it!

**Comment by erm** [comment]
Ah, no, it is not my intention to include any hard dependencies in this package. There used to be a check for this that I removed temporarily while doing a refactor, but forgot to re-add it. I'll push a new release later that makes this optional for HTTP events.

Thanks for bringing this to my attention! :)

**Comment by erm** [comment]
Included in 0.9.2.
